hkmc2/shared/src/main/scala/hkmc2/utils/utils.scala

```diff
@@ -36,11 +37,16 @@ extension (t: Product)
       case Nil => "Nil"
       case xs: List[_] => "Ls of \n" + xs.iterator.map(aux(_)).mkString("\n").indent("  ")
       case s: String => s.escaped
-      case FldFlags(mut, spec, genGetter) =>
+      case TermDefFlags(mod) =>
+        val flags = Buffer.empty[String]
+        if mod then flags += "module"
+        if flags.isEmpty then "()" else flags.mkString("(", ", ", ")")
+      case FldFlags(mut, spec, genGetter, mod) =>
         val flags = Buffer.empty[String]
         if mut then flags += "mut"
         if spec then flags += "spec"
         if genGetter then flags += "gen"
+        if mod then flags += "module"
```

Comment:
At this point you'd better just use a `StringBuilder` instead of an intermediate collection.